doc/source/whatsnew/v0.22.0.txt

```diff
@@ -22,7 +22,7 @@ New features
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
--
+- `to_csv()` and `to_json()` now support 'infer' as `compression` argument (was already supported by `to_pickle()` before)
```

Comment:
``as a ``compression=`` kwarg``. you don't need to mention pickle. add the issue number.

use ``:func:``~DataFrame.to_csv` `` and similar to ``to_json``

Reply:
can you add tests for to_json?